ecgtools/parsers/cesm.py

```diff
@@ -44,6 +45,21 @@
     'ww3.h': {'component': 'wav'},
 }
 
+frequency_dict = {
+    'cam.h0': 'month_1',
+    'cam.h1': 'day_1',
+    'cam.h2': 'hour_6',
+    'cice.h': 'month_1',
+    'cice.h1': 'day_1',
+    'clm2.h0': 'month_1',
+    'clm2.h1': 'day_1',
+    'pop.h.ecosys.nyear1': 'year_1',
+    'pop.h.ecosys.nday1': 'day_1',
+    'pop.h': 'month_1',
+    'rtm.h0': 'month_1',
+    'rtm.h1': 'day_1',
+}
```

Comment:
Instead of recording this as a separate dict, could we merge this with the stream dict as we had it before. The main idea would be to try getting the frequency from the dataset attribute, and fall back to the default value in the stream dict when we are unable to get the frequency from `time_period_freq` attribute. What do you think?

Reply:
I think we can keep this separate - unless there would be a way to plug in a way for the user to specify this dictionary separately? I think we should allow the user to input a dictionary with the stream frequency for this...

Reply:
Sounds good 👍🏽